packages/syntax-floating-components/package.json

```diff
@@ -21,6 +21,8 @@
     "test:watch": "vitest"
   },
   "dependencies": {
+    "@ariakit/react": "^0.3.0",
```

Comment:
@benjroy could we make sure that we use the latest version of @ariakit/react v0.3.3